apps/arweave/src/ar_nonce_limiter_server_sup.erl

```diff
@@ -21,7 +21,7 @@ start_link() ->
 %% ===================================================================
 
 init([]) ->
-	case ar_config:is_vdf_server() of
+	case ar_config:is_vdf_server() andalso not ar_config:verify() of
```

Comment:
I think, it makes more sense to check for auto_join()

Reply:
Latest commit removes this check and instead uses the `set_dependent_flags` process to disable all vdf features.